airflow/executors/kubernetes_executor.py

```diff
@@ -834,7 +429,7 @@ def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[Task
         tis_to_flush_by_key = {ti.key: ti for ti in tis if ti.queued_by_job_id}
         kube_client: client.CoreV1Api = self.kube_client
         for scheduler_job_id in scheduler_job_ids:
-            scheduler_job_id = pod_generator.make_safe_label_value(str(scheduler_job_id))
+            scheduler_job_id = self._make_safe_label_value(str(scheduler_job_id))
```

Comment:
the `str` calls stand out.  i wonder why they are there.  perhaps it's something that should be pushed down.  e.g. maybe signature should accept `int` and wrap with `str` in that case.

Reply:
Yeah, I noticed that as well, I'd like to minimize the functional changes in this PR though